docs/document/content/dev-manual/encrypt.en.md

```diff
@@ -5,20 +5,22 @@ weight = 12
 chapter = true
 +++
 
-## SPI Interface
+## EncryptAlgorithm
 
-| SPI Name         | Description            |
-| ---------------- | ---------------------- |
-| EncryptAlgorithm | Data encrypt algorithm |
+### Fully-qualified class name
 
-## Sample
+[`org.apache.shardingsphere.encrypt.algorithm`](https://github.com/apache/shardingsphere/blob/master/shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-api/src/main/java/org/apache/shardingsphere/encrypt/spi/EncryptAlgorithm.java)
```

Comment:
Only package name is not enough, please add class name

Reply:
how to get that?

Reply:
For this case, it should be `org.apache.shardingsphere.encrypt.spi.EncryptAlgorithm`